src/main/java/io/appium/java_client/MultiTouchAction.java

```diff
@@ -80,9 +81,8 @@ public MultiTouchAction perform() {
         ImmutableList.Builder<Object> listOfActionChains = ImmutableList.builder();
         ImmutableList<TouchAction> touchActions = actions.build();
 
-        touchActions.forEach(action -> {
-            listOfActionChains.add(action.getParameters().get("actions"));
-        });
+        touchActions.forEach(action ->
```

Comment:
why not to replace it with something like `ImmutableList.Builder<Object> listOfActionChains = ImmutableList.copyOf(touchActions.stream().map(x -> x.getParameters().get("actions")));` ?